db/migrate/20250107094042_update_participant_declaration_ecf_types.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UpdateParticipantDeclarationECFTypes < ActiveRecord::Migration[7.1]
+  def up
+    declarations = ParticipantDeclaration::ECF.joins(:participant_profile)
+    mentor_declarations = declarations.where(participant_profile: { type: "ParticipantProfile::Mentor"})
```

Comment:
How about adding a scope to declarations? I can see this `where type...` will be duplicated a lot
ie: `declarations.for_mentors`

Reply:
I think a scope in this scenario may confuse things as it has to look at the participant profiles for the migration to work, but with the changes we're bringing in we wouldn't want that to be used in other areas of the application as the new types should be the used 🤔